scripts/wards.sh

```diff
@@ -3,7 +3,18 @@ set -e
 
 [[ "$ETH_RPC_URL" ]] || { echo "Please set a ETH_RPC_URL"; exit 1; }
```

Comment:
```suggestion
[[ "$ETH_RPC_URL" ]] || { echo "Please set a ETH_RPC_URL"; [ -v PS1 ] && return || exit; }
```

Avoids closing the terminal window when running the script directly.

Reply:
Can also use on next line.

Reply:
Avoids closing the terminal window when running the script directly (e.g. `. ./scripts/wards.sh`)

Reply:
I've reverted latest changes for now, as discussed in chat, to investigate more on an approach to address this issue.